cmd/archive.go

```diff
@@ -12,6 +12,7 @@ type cmdArchive struct {
 	gs *state.GlobalState
 
 	archiveOut     string
+	archiveStdout  bool
```

Comment:
I'm fine with either! I went for this because I think it's the most unix conventional thing, but I can also see the consistency argument.

Changing it is trivial, so I'll leave the decision to you :)

Reply:
@grafana/k6-core, any opinions about which way to go? :thinking:

Reply:
I am for `-O -` to output to stdout instead of to a file `-`

Reply:
It is fine to me :+1:

Reply:
Same here ✋🏻 But please make sure to explicitly document this behavior in the flag description and in the docs website 🙇🏻

Reply:
Same! 👍🏻